src/main/java/pages/ProfilePage.java

```diff
@@ -0,0 +1,66 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * This class contains the webelements of the register form
+ */
+public class ProfilePage extends SignUpPage {
```

Comment:
I think this class should not extend SignUpPage

Reply:
Because in this class is the webelement in common userName. :)